<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <!-- Dataset -->
  <arg name="data_path" default="/media/lukas/T7/data/clouds_20m/shopville_2_none/eval_clouds.csv" />  <!-- hauptgebaeude niederdorf station shopville -->



  <!-- Cloud Visualizer -->
  <node name="cloud_visualizer" pkg="lidar_motion_detection_ros" type="cloud_visualizer" output="screen" args="--alsologtostderr" required="true">
    <param name="file_path" value="$(arg data_path)" />
    <param name="static_point_scale" value="0.05" />
    <param name="dynamic_point_scale" value="0.1" />
    <param name="out_of_bounds_color" value="[0.8,0.8,0.8,1]" />
  </node>
  
  <!-- Visualize -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_motion_detection_ros)/config/rviz/cloud_visualizer.rviz"/>

</launch>
